src/components/TextField/TextField.tsx

```diff
@@ -125,6 +130,16 @@ class TextField extends React.PureComponent<Props, State> {
         />
       )
       : null;
+    
+    var counterTextMarkup;
+    if(enableTextCouter == true)
+    {
+      var maxLengthString = '';
+      if(maxLength)
+        maxLengthString = "/" + maxLength;
```

Comment:
Use ternary syntax for maxLengthString: 
    const maxLengthString = maxLength ? ``/${maxLength}` : '';